### PR TITLE
Add support for multiply languages

### PR DESCRIPTION
**Pull request by sangyye**
I added support for more languages than english, so it's easy to add other languages. The detection works via the default language in the browser. If this is not detected or the language is not translated, it defaults to english.

Limitations:
- The language chooser only works within the root of the domain, no folders are supported. (but the Stylesheets work)

TODO:
- [ ] Add a language chooser

### CODE REVIEW

**Comment by sangyye** [comment]
Cannot really reproduce the travis error, everything looks fine on my machine:

```
$ bundle exec htmlproof --check-html --disable-external ./_site
Running ["ImageCheck", "ScriptCheck", "HtmlCheck", "LinkCheck"] on ./_site on *.html...


Ran on 2 files!


HTML-Proofer finished successfully.
$ bundle exec htmlproof -v
htmlproof 2.6.4
```

**Comment by infroware** [comment]
I think it's better to attach the files and send it to you , because I have
no experience

**Comment by sangyye** [comment]
@khaledalhasan I think you can't add files to github comments via email.

**Comment by jhilden** [comment]
thanks for the PR @sangyye 

@khaledalhasan how about introducing arabic together at the next meeting?  The next meeting will be Thursday next week (March 3rd).  Will you be able to come?  If yes, it would be nice if you could fill out the Doodle here http://doodle.com/poll/vspxtbxfvfxwpdtg

**Comment by infroware** [comment]
Yes , of course I'll be there , I filled out the doodle also .

**Comment by sangyye** [comment]
I removed the `--check-html` until I know why I get the strange error in travis but not on my local machine

**Comment by sangyye** [comment]
I have no idea why this gives an error. Everything fine on my end and on the [online-validator](https://validator.w3.org/nu/?doc=http%3A%2F%2Fsangyye.github.io%2Frefugees%2F).

**Comment by jhilden** [comment]
@sangyye since we have no checked in Gemfile.lock, Travis might be installing different versions of our dependencies.  Consider comparing the versions, and I don't think there is any reason not to check in the Gemfile.lock for this project.

**Comment by sangyye** [comment]
@jhilden Okay, I added it, but I already checked the versions, and it seems that there are no difference in the versions.

**Comment by sangyye** [comment]
@jhilden Even with the added Gemfile.lock it errors on travis and works fine on my local computer. Can't really understand that...

File: _layouts/default.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
+    <link rel="stylesheet" type="text/css" href="{{ site.github.url }}/stylesheets/stylesheet.css" media="screen">
```
Comment (jhilden) [inline _layouts/default.html:8]: why is this necessary?  are relative paths not working here anymore?

File: .travis.yml [hunk 2]
```diff
@@ -7,9 +7,8 @@ branches:
   only:
     - gh-pages
 
-rvm:
-  - 2.2.3
-script: bundle exec jekyll build && htmlproof --check-html --disable-external ./_site
+rvm: 2.2.3
+script: bundle exec jekyll build && bundle exec htmlproofer --check-html --disable-external ./_site --file-ignore ./_site/index.html
```
Comment (jhilden) [inline .travis.yml:11]: could you add a comment why exactly we are ignoring that one file?

File: _layouts/default.html (hunk 1 above)
Comment (sangyye) [inline _layouts/default.html:8]: no, because I use redirects on subfolders with the language, you need absolute paths for the stylesheets. Otherway the index in the folder doesn't find the sheets.

File: .travis.yml (hunk 2 above)
Comment (sangyye) [inline .travis.yml:11]: Because I don't get the error on any of my machines, linux or mac after testing for a whole weekend. I couldn't find it. Even online checker say, this file is okay, so i ignore an error on travis I can't reproduce or understand.
